pulsar-broker-common/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGeneratorUtils.java

```diff
@@ -67,14 +68,21 @@ public static void generateSystemMetrics(SimpleTextOutputStream stream, String c
                 stream.write(sample.name);
                 if (!sample.labelNames.contains("cluster")) {
```

Comment:
Maybe we can add this `stream.write("{");` before this check. The char `{` is necessary and it's easier to understand.

Reply:
That's a good idea. The code has been changed